examples/appliedenergistics2.groovy

```diff
@@ -0,0 +1,90 @@
+
+// Can be access via either `appliedenergistics2` or `ae2`
+
+// Bracket Handlers
+
+// Get the P2P Tunnel type from the enum. Case insensitive.
+tunnel('me')
```

Comment:
I wonder if it would be better to add something like this to the mod compat class so it would look like 'mods.ae2.tunnel('me')'

Reply:
at the very least having that as an alternate way to get bracket handlers would be a good idea.